script.py

```diff
@@ -0,0 +1,5 @@
+from scapy.all import *
```

Comment:
I don't think this script should be pushed to the repo, I'm assuming it's used for testing only?

Reply:
Any plan to address this?